src/iss.js

```diff
@@ -295,6 +300,7 @@ export class Service {
         }
 
         var request_: searchRequest = {
+            q: "",
```

Comment:
You want to define `q` as optional. Sending `q` in the query will change what happens.